### PR TITLE
update: updated README.md

### PR DESCRIPTION
**Pull request by redconOne**
# Update README.md

1. Updated badges
2. Update install instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,31 +111,34 @@ npm install npm@latest -g
...
+   ```sh
+   npm run start
    ```
 
 ## Usage
 
-- How to
-- Use
-- Atelier
+Atelier is runs on the designated port and opens a webpage as soon as the npm run start command is utilized. It can also be accessed utilizing localhost:PORT directly in the browser.
```
Comment (1Oulala) [inline README.md:139]: Hi Ming,
I am going to assume you meant  " Atelier run on..  " and not " Atelier is runs ... "

**Review by 1Oulala — COMMENTED** [review]
Everything else loos good but i think we have a minor typo on line 139 in the README.md

**Comment by redconOne** [comment]
@1Oulala typo has been corrected.

**Review by 1Oulala — APPROVED** [review]
Love it!

**Pull request merged**